network-api/networkapi/wagtailpages/tests/test_blog.py

```diff
@@ -860,3 +860,27 @@ def test_authors_detail_non_existent_id_argument(self):
         )
         response = self.client.get(path=blog_author_url)
         self.assertEqual(response.status_code, 404)
+
+
+class TestBlogPageTopics(test.TestCase):
+
+    def test_factory(self):
+        blog_factories.BlogPageTopicFactory()
+
+    def test_get_topics_method(self):
+        # Clearing test instance of any existing topics
+        blog_models.BlogPageTopic.objects.all().delete()
+
+        test_topic = blog_factories.BlogPageTopicFactory(name="Test_Topic_1")
+        blog_factories.BlogPageTopicFactory(name="Test_Topic_2")
+        blog_factories.BlogPageTopicFactory(name="Test_Topic_3")
+
+        # Creating a list of all created BlogPageTopics sorted by name,
+        # with an additional option of "All".
+        list_of_sorted_topics = [('All', 'All')]
+        for topic in blog_models.BlogPageTopic.objects.all().order_by('name'):
+            list_of_sorted_topics.append((topic.name, topic.name))
```

Comment:
What do you think of just spelling this out manually? I find it often a bit hard to understand a test when there is a lot of automation in the setup. Also, this looks basically like the implementation 😅

Reply:
Hi Tibor, after giving this a second look I agree that just having the list defined and shown in the order it should be in is a little more clear then doing the sorting and arranging this way, I have updated the test!